content/src/Environment.ts

```diff
@@ -33,10 +33,11 @@ export const DEFAULT_ETH_NETWORK = 'ropsten'
 export const DEFAULT_DCL_PARCEL_ACCESS_URL_ROPSTEN =
   'https://api.thegraph.com/subgraphs/name/decentraland/land-manager-ropsten'
 export const DEFAULT_DCL_PARCEL_ACCESS_URL_MAINNET = 'https://api.thegraph.com/subgraphs/name/decentraland/land-manager'
-export const DEFAULT_DCL_COLLECTIONS_ACCESS_URL_ROPSTEN =
-  'https://api.thegraph.com/subgraphs/name/decentraland/collections-ropsten'
 export const DEFAULT_DCL_COLLECTIONS_ACCESS_URL_MAINNET =
-  'https://api.thegraph.com/subgraphs/name/decentraland/collections'
+  'https://thegraph.com/explorer/subgraph/decentraland/collections-ethereum-mainnet'
```

Comment:
For .org we are going to support `ethereum mainnet` and `matic mainnet` to get all the wearables a user has. 

What do you think of having `DEFAULT_DCL_COLLECTIONS_ACCESS_URL_ETHEREUM_MAINNET`
`DEFAULT_DCL_COLLECTIONS_ACCESS_URL_ETHEREUM_ROPSTEN`
`DEFAULT_DCL_COLLECTIONS_ACCESS_URL_MATIC_MAINNET`
`DEFAULT_DCL_COLLECTIONS_ACCESS_URL_MATIC_MUMBAI`
etc

?

Reply:
Also, IDK if it was clear but at that time, collections will be fetched from `ethereum mainnet` and `matic mainnet` for .org and `ethereum mumbai` for .zone. 

Later, we will add `ethereum goerli` for .zone too.